name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/AbsolutePerformancePieChart.java

```diff
@@ -0,0 +1,334 @@
+        data.percentageString = Values.Percent2.format((wrapper.invested / (double) total));
+        data.value = Values.Money.format(Money.of(snapshot.getCurrencyCode(), wrapper.invested));
+        id2nodeData.put(Messages.LabelAbsoluteInvestedCapital, data);
+
+
+        values.add(wrapper.delta / Values.Amount.divider());
+        labels.add(Messages.LabelAbsoluteDelta);
+
+        data = new NodeData();
+        data.label = Messages.LabelAbsoluteDelta;
+        data.percentageString = Values.Percent2.format((wrapper.delta / (double) total));
+        data.value = Values.Money.format(Money.of(snapshot.getCurrencyCode(), wrapper.delta));
+        id2nodeData.put(Messages.LabelAbsoluteDelta, data);
+
+
+        ICircularSeries<?> circularSeries = (ICircularSeries<?>) chart.getSeriesSet()
+                        .getSeries(Messages.ColumnAbsolutePerformance_MenuLabel);
+        if (circularSeries == null)
+        {
+            circularSeries = createSeries(values, labels);
+        }
+        else
+        {
+            // refresh values only for smoother update
+            ListIterator<String> iter = labels.listIterator();
+            while (iter.hasNext())
+            {
+                int idx = iter.nextIndex();
+                String label = iter.next();
+                Node node = circularSeries.getNodeById(label);
+                if (node != null)
+                {
+                    node.setValue(values.get(idx));
+                }
+            }
+
+        }
+
+        setColors(circularSeries, values.size(), wrapper.delta > 0);
+
+        chart.updateAngleBounds();
+        chart.redraw();
+    }
+
+    private void countAssets(InvestedWrapper wrapper, DataSeriesSet dataSeriesSet)
+    {
+        snapshot.getAssetPositions() //
+                        .filter(p -> p.getValuation().getAmount() > 0) //
+                        .forEach(p -> {
+                            String nodeId = p.getInvestmentVehicle().getUUID();
+                            
+                            var dataSeries = dataSeriesSet.getAvailableSeries().stream()
+                                            .filter(ds -> ds.getType() != Type.ACCOUNT && ds.getUUID().contains(nodeId))
+                                            .findAny().orElse(null);
+    
+                            if (dataSeries != null)
+                            {
+                                var perf = cache.lookup(dataSeries,
+                                                Interval.of(LocalDate.ofYearDay(1900, 1), LocalDate.now()));
+    
+                                long[] d = perf.calculateAbsoluteInvestedCapital();
+                                wrapper.invested += d.length > 0 ? d[d.length - 1] : 0L;
+    
+                                long[] d2 = perf.calculateAbsoluteDelta();
+                                wrapper.delta += d2.length > 0 ? d2[d2.length - 1] : 0L;
+    
+                                position = p.getDescription();
```

Comment:
Actually label is used only when it loops over 1 item (but it is not known beforehand), if there is more items, portfolio name is used as label instead. 
But yes it would be better to have loop label variable in `wrapper` temp object.